src/libcore/num/mod.rs

```diff
@@ -44,7 +44,9 @@ pub struct Wrapping<T>(#[stable(feature = "rust1", since = "1.0.0")] pub T);
 pub mod wrapping;
 
 // All these modules are technically private and only exposed for libcoretest:
```

Comment:
It may be a good idea to also cfg' out the corresponding tests, otherwise those will fail.